README.md

```diff
@@ -13,3 +13,51 @@ All metric definitions can and should be referenced in other tooling and their c
 - **Looker Explore** specific configs or defaults can be added in the `looker/` directory. This directory also supports the definitions of "statistics" (i.e. specific aggregations of metric-hub metrics), which will be materialized as measures in the autogenerated Explores. How to do this is explained in detail in the [documentation](https://docs.telemetry.mozilla.org/concepts/metric_hub.html#using-metrics-in-looker).
 
 Tool-specific configs take precedence over the metric definitions in the top level `definitions/` folder when used in the tooling, while in all other contexts `definitions/` is seen as the source of truth.
+
+
+## Experiment Analysis Considerations
+
+metric-hub's CI is configured to detect when a given change would affect experiment analysis, and to automatically rerun analysis for these experiments in certain scenarios. This behavior is not always desirable, and so it can also be overridden using `[ci rerun-skip]`. Continue reading to learn more about the automated rerun scenarios and when you may want to use `[ci rerun-skip]`.
+
+### Automated Analysis Reruns
+
+There are two main scenarios that trigger analysis reruns:
+
+1. Changing default metrics (i.e., in `jetstream/defaults`)
+    * this will trigger a rerun in the affected application of:
+      * **all live** experiments
+      * **completed** experiments that have **ended within the past 90 days**
```

Comment:
TIL, I thought it was only live experiments.

Reply:
Should we change it to live experiments only?

Reply:
IMO, yeah, we should.  I think we tend to follow a pattern of "avoid reanalyzing and thus potentially changing past experiment results unless explicitly requested" and this seems to break that.

Reply:
Yea unless I'm mistaken (and it is late on a Friday, so...) the check for that is here: https://github.com/mozilla/jetstream/blob/main/jetstream/cli.py#L414-L415.

Since this function is used by more than just the CI rerun functionality, we should make sure this won't affect anything else before changing, but in general it should be pretty straightforward to do.